definitions/ext-cdn_generic/dashboard.json

```diff
@@ -0,0 +1,147 @@
+{
+  "name": "CDN Health Metrics",
+  "description": null,
+  "permissions": "PUBLIC_READ_WRITE",
```

Comment:
The PR is failing because of the dashboard, you can fix it by sanitizing it, for example using:

```
docker-compose run sanitize-dashboards
```